packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -108,9 +98,11 @@ class Nav extends React.Component {
   }
 
   render() {
-    const { quickLinks, navItems, profileLinks, userName, avatar, signoutUrl, logo, isMobileNavOpen, onRequestClose, ...customProps } = this.props;
+    const { quickLinks, navItems, profile, logo, isMobileNavOpen, onRequestClose, ...customProps } = this.props;
     const profileId = 'profile-popup-button';
 
+    const willRenderProfile = profile.userName || profile.avatar || profile.signinUrl;
```

Comment:
do we have some use case where we aren't rendering the profile?

Reply:
yes we had this as a new requirement:
`✓ - if sign-in url provided and avatar/user NOT provided, Profile link should just be a sign-in link. 
✓ -- If no sign-in url then do not render component at all `